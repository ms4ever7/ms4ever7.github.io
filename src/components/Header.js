
import React from 'react';

const Header = () => <header className='header layout-column layout-align-start-center'>
  <h1 className='header__title'>Royal Shoes</h1>
  <hr className='header__separate-line'/>
  <h3 className='header__subtitle'>Choose your dream and get up</h3>
</header>

export default Header;