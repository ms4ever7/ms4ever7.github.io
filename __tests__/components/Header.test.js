import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Header from '../../src/components/Header';

describe('Header', () => {
  let wrapper;


  it('should render correctly', () => {
    wrapper = shallow(
      <Header />
    );

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('has appropriate classnames', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.find('.header').exists()).toBeTruthy();
    expect(wrapper.find('.header__title').exists()).toBeTruthy();
    expect(wrapper.find('.header__separate-line').exists()).toBeTruthy();
    expect(wrapper.find('.header__subtitle').exists()).toBeTruthy();
  });
});
