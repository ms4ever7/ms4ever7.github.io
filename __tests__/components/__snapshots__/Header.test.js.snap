// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExploreStrainsList should render correctly 1`] = `
<header
  className="header layout-column layout-align-start-center"
>
  <h1
    className="header__title"
  >
    Royal Shoes
  </h1>
  <hr
    className="header__separate-line"
  />
  <h3
    className="header__subtitle"
  >
    Choose your dream and get up
  </h3>
</header>
`;

exports[`Header should render correctly 1`] = `
<header
  className="header layout-column layout-align-start-center"
>
  <h1
    className="header__title"
  >
    Royal Shoes
  </h1>
  <hr
    className="header__separate-line"
  />
  <h3
    className="header__subtitle"
  >
    Choose your dream and get up
  </h3>
</header>
`;
